apps/theming-designer/src/components/FabricPalette.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import { FabricSlots, IThemeRules } from 'office-ui-fabric-react/lib/ThemeGenerator';
+import { Stack } from '../../../../packages/office-ui-fabric-react/lib/index';
+import { mergeStyles } from '@uifabric/merge-styles';
+
+export interface IFabricPaletteProps {
+  themeRules: IThemeRules | undefined;
```

Comment:
Probably best not to accept `undefined` and remove `!` from usages of `themeRules`.